src/System.Private.CoreLib/shared/System/Diagnostics/ConditionalAttribute.cs

```diff
@@ -2,16 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
+
 namespace System.Diagnostics
 {
     [AttributeUsage(AttributeTargets.Method | AttributeTargets.Class, AllowMultiple = true)]
     public sealed class ConditionalAttribute : Attribute
     {
-        public ConditionalAttribute(string conditionString)
+        public ConditionalAttribute(string? conditionString)
```

Comment:
We should make both this and the property be non-nullable.  While technically you could programmatically pass null and nothing would complain, the attribute is useless if you do that, and the C# compiler will actually error if you try to do `[Conditional(null)]`.